src/barotropicqg.jl

```diff
@@ -397,4 +431,16 @@ Returns the extraction of domain-averaged energy by drag μ.
   return params.μ / (grid.Lx * grid.Ly) * parsevalsum(vars.uh, grid)
 end
 
+"""
+    drag_ens(prob)
+
+Returns the extraction of domain-averaged enstrophy by drag/hypodrag μ.
+"""
+@inline function drag_ens(prob)
```

Comment:
A good name for this function might be `drag_enstrophy_dissipation`. In light of this function, the first function could be called `viscous_enstrophy_dissipation`.

Reply:
I like your suggestions of `enstrophy_dissipation`, `enstrophy_work` and, for consistency, `enstrophy_drag`. This is consistent with the diagnostics for energetics; `dissipation`, `work` and `drag` (rather than specifying viscous dissipation and drag dissipation). 

If we took your later suggestions the energetics diagnostics should probably be changed too.